x-pack/test/api_integration/apis/cloud_security_posture/status/status_waiting_for_results.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import type { CspSetupStatus } from '@kbn/cloud-security-posture-plugin/common/types';
+import { setupFleetAndAgents } from '../../../../fleet_api_integration/apis/agents/services';
+import { generateAgent } from '../../../../fleet_api_integration/helpers';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { createPackagePolicy } from '../helper';
+
+export default function (providerContext: FtrProviderContext) {
+  const { getService } = providerContext;
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('GET /internal/cloud_security_posture/status', () => {
+    let agentPolicyId: string;
+
+    describe('STATUS = WAITING_FOR_RESULT TEST', () => {
+      setupFleetAndAgents(providerContext);
+
+      beforeEach(async () => {
+        await kibanaServer.savedObjects.cleanStandardList();
+        await esArchiver.load('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+        const getPkRes = await supertest
+          .get(`/api/fleet/epm/packages/fleet_server`)
+          .set('kbn-xsrf', 'xxxx')
+          .expect(200);
+        const pkgVersion = getPkRes.body.item.version;
+        await supertest
+          .post(`/api/fleet/epm/packages/fleet_server/${pkgVersion}`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({ force: true })
+          .expect(200);
+
+        const { body: agentPolicyResponse } = await supertest
+          .post(`/api/fleet/agent_policies`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({
+            name: 'Test policy a1',
+            namespace: 'default',
+          });
+
+        agentPolicyId = agentPolicyResponse.item.id;
+
+        await supertest
+          .post(`/api/fleet/fleet_server_hosts`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({
+            id: 'test-default-a1',
+            name: 'Default',
+            is_default: true,
+            host_urls: ['https://test.com:8080', 'https://test.com:8081'],
+          })
+          .expect(200);
+        await generateAgent(providerContext, 'healthy', `Agent policy test 2`, agentPolicyId);
+      });
+
+      afterEach(async () => {
+        await kibanaServer.savedObjects.cleanStandardList();
+        await esArchiver.unload('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+      });
+
+      it(`Should return waiting_for_result when installed kspm, has no findings and it has been more than 10 minutes since the installation`, async () => {
```

Comment:
i think this should say "less than 10 minutes" no? also worth mentioning that it has findings in the regular index, not the latest index. and lastly, maybe worth adding a 9 minutes after installation or something to actually test the timeframe that we display that message.